### PR TITLE
Added code to resize columns to fit data.

### PR DESCRIPTION
**Pull request by cienvaras**
Fix for this: https://github.com/NuCivic/ok_data/issues/134

### CODE REVIEW

File: js/visualization_entity_tables_edit.js [hunk 1]
```diff
@@ -1,13 +1,25 @@
...
+
     var $resourceField = $('#edit-field-uuid-resource-und-0-target-uuid');
+    var $resizeField = $('#edit-field-ve-table-resize-und');
+    var $settingsField = $('#edit-field-ve-settings-und-0-value');
 
     $resourceField.on('autocompleteSelect', function(event, node) {
       var re = /\[(.*?)\]/;
       var uuid = re.exec($resourceField.val())[1];
-      $('#edit-field-ve-settings-und-0-value').val(JSON.stringify({source:'/node/' + uuid + '/download'}));
+      settings.source = '/oklahoma/node/' + uuid + '/download';
```
Comment (topicus) [inline js/visualization_entity_tables_edit.js]: We need to remove this before merge to prevent this library remain tied to the OK project.

**Comment by topicus** [comment]
Could you provide a briefly explanation with the steps I need to follow in order to make this work? I've cloned and switched to this branch but after that I couldn't see any option in the table's creation page.

However I've got an error in the chrome console: 
add:1 Uncaught SyntaxError: Unexpected end of input

I did a drush fr visualization_entity_tables just in case would be an eck update but without succes.

File: js/visualization_entity_tables_edit.js (hunk 1 above)
Comment (cienvaras) [inline js/visualization_entity_tables_edit.js]: Ok, fixed.

**Comment by cienvaras** [comment]
@topicus I just added a new field to the bundle and removed some unused elements, it should work with a simple feature revert.

I can't reproduce the error you're getting. It might be related with the checkbox not being added to the bundle.

File: js/visualization_entity_tables_edit.js [hunk 2]
```diff
@@ -1,13 +1,25 @@
 (function($) {
+
   $(document).on('ready', function(){
+
+    var settings = JSON.parse($('#edit-field-ve-settings-und-0-value').val());
```
Comment (topicus) [inline js/visualization_entity_tables_edit.js]: Here you are assuming that $('#edit-field-ve-settings-und-0-value').val() is an object, that was causing the end of input error. You need to check that before the parse.

Comment (cienvaras) [inline js/visualization_entity_tables_edit.js]: Added exception handling to prevent this error.

**Pull request merged**